Compilation fails on OSX

Hi there,

We are trying to compile plotjuggler for ROS on Conda (https://github.com/RoboStack/ros-noetic). We used to compile it just fine, however with the recent builds this error occurs for our OSX builds:

```
[30/306] Building CXX object 3rdparty/Qt-Advanced-Docking/CMakeFiles/qt_advanced_docking.dir/src/linux/FloatingWidgetTitleBar.cpp.o
FAILED: 3rdparty/Qt-Advanced-Docking/CMakeFiles/qt_advanced_docking.dir/src/linux/FloatingWidgetTitleBar.cpp.o 
$BUILD_PREFIX/bin/x86_64-apple-darwin13.4.0-clang++ -DADS_STATIC -DCOMPILED_WITH_CATKIN -DPJ_MAJOR_VERSION=\"3\" -DPJ_MINOR_VERSION=\"0\" -DPJ_PATCH_VERSION=\"4\" -DQT_CORE_LIB -DQT_GUI_LIB -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DROS_BUILD_SHARED_LIBS=1 -I3rdparty/Qt-Advanced-Docking -I$SRC_DIR/ros-noetic-plotjuggler/src/work/3rdparty/Qt-Advanced-Docking -I3rdparty/Qt-Advanced-Docking/qt_advanced_docking_autogen/include -I$PREFIX/include -I$PREFIX/include/qt -I$PREFIX/include/qt/QtCore -I$PREFIX/./mkspecs/macx-clang -I$PREFIX/include/qt/QtWidgets -I$PREFIX/include/qt/QtGui -I/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/System/Library/Frameworks/OpenGL.framework/Headers -I$PREFIX/include/qt/QtConcurrent -I$PREFIX/include/qt/QtPrintSupport -I$PREFIX/include/qt/QtXml -I$PREFIX/include/qt/QtWebSockets -I$PREFIX/include/qt/QtNetwork -I$PREFIX/include/qt/QtSvg -I$PREFIX/include/qt/QtOpenGL -I$SRC_DIR/ros-noetic-plotjuggler/src/work/./include -I$SRC_DIR/ros-noetic-plotjuggler/src/work/./3rdparty -I$SRC_DIR/ros-noetic-plotjuggler/src/work/./3rdparty/color_widgets/include -I$SRC_DIR/ros-noetic-plotjuggler/src/work/./3rdparty/Qt-Advanced-Docking/include -I$SRC_DIR/ros-noetic-plotjuggler/src/work/./3rdparty/paho.mqtt.c/src -I$SRC_DIR/ros-noetic-plotjuggler/src/work/./3rdparty/qwt/src -I$SRC_DIR/ros-noetic-plotjuggler/src/work/./3rdparty/sol -I$SRC_DIR/ros-noetic-plotjuggler/src/work/./3rdparty/lua-5.4.1/src -I$SRC_DIR/ros-noetic-plotjuggler/src/work/3rdparty/Qt-Advanced-Docking/include/Qads -fdiagnostics-color=always -DBOOST_ERROR_CODE_HEADER_ONLY -g -fno-omit-frame-pointer  -fPIC -O3 -DNDEBUG -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk -mmacosx-version-min=10.9 -fPIC -std=c++14 -MD -MT 3rdparty/Qt-Advanced-Docking/CMakeFiles/qt_advanced_docking.dir/src/linux/FloatingWidgetTitleBar.cpp.o -MF 3rdparty/Qt-Advanced-Docking/CMakeFiles/qt_advanced_docking.dir/src/linux/FloatingWidgetTitleBar.cpp.o.d -o 3rdparty/Qt-Advanced-Docking/CMakeFiles/qt_advanced_docking.dir/src/linux/FloatingWidgetTitleBar.cpp.o -c $SRC_DIR/ros-noetic-plotjuggler/src/work/3rdparty/Qt-Advanced-Docking/src/linux/FloatingWidgetTitleBar.cpp
$SRC_DIR/ros-noetic-plotjuggler/src/work/3rdparty/Qt-Advanced-Docking/src/linux/FloatingWidgetTitleBar.cpp:197:34: error: too many arguments to function call, expected 0, have 1
                        d->FloatingWidget->showNormal(true);
                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^~~~
$PREFIX/include/qt/QtWidgets/qwidget.h:482:10: note: 'showNormal' declared here
    void showNormal();
         ^
1 error generated.
```

Is it safe to just remove the `true` argument? Is there something which needs fixing here? I am not sure whether it's specific to the conda build.

Many thanks!

--- a/3rdparty/Qt-Advanced-Docking/CMakeLists.txt
+++ b/3rdparty/Qt-Advanced-Docking/CMakeLists.txt
@@ -52,7 +52,7 @@ set(ads_SRCS
     include/Qads/DockComponentsFactory.h
 )
 
-if (UNIX)
+if (UNIX AND NOT APPLE)
     set(ads_SRCS
         src/linux/FloatingWidgetTitleBar.cpp
         ${ads_SRCS}

